mozilla-mobile/focus-android/app/src/main/res/values/strings.xml

```diff
@@ -784,5 +784,5 @@ The new line here must be kept as the second half of the string is clickable for
 
     <!-- Text for context menu additional note.
     %1$s is a placeholder for the app name. -->
-    <string name="contextmenu_erased_images_note">Saved and shared images &lt;b&gt;will not be&lt;/b&gt; deleted when you erase %1s history</string>
+    <string name="contextmenu_erased_images_note">Saved and shared images &lt;b&gt;will not be&lt;/b&gt; deleted when you erase %1$s history</string>
```

Comment:
@jonalmeida Could you please change the string ID here? I know it's going to invalidate the string, but I'm afraid many localizers are going to keep the placeholder "wrong" in their localizations otherwise, and that would break import. Thanks! :)